pkg/operator/operator.go

```diff
@@ -94,6 +95,7 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 	go o.manager.Start(ctx.Done())
 	go logLevelController.Run(ctx, 1)
 	go managementStateController.Run(ctx, 1)
+	go connectivitycheck.Start(ctx, o.ccfg.KubeConfig)
```

Comment:
it looks like this already returns, so no need to run it in a goroutine.

Reply:
Done